pkg/watch/health.go

```diff
@@ -156,13 +156,15 @@ func updatePods(
 
 		// if a manifest is in reality but not current a podwatch is created
 		// with that manifest and added to newCurrent
-		if missing && man.Manifest.StatusPort != 0 {
+		if missing {
 			sc := StatusChecker{
 				ID:     man.Manifest.Id,
 				Node:   node,
 				Client: client,
 			}
-			if man.Manifest.StatusHTTP {
+			if man.Manifest.StatusPort == 0 {
+				sc.URI = ""
```

Comment:
I don't even follow what you're suggesting.  But since it's a bad idea, we probably don't want to do it anyway.